Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -1,56 +1,56 @@
-using System;
```

Comment:
This file is just line endings right?
I think Stef changed them lol. I saw a MacOS line endings warning the other day.
But yeah try to use ```git config --global core.autocrlf false``` if you are not already.